packages/config/lib/options.ts

```diff
@@ -163,26 +185,43 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     name: 'includeShadowDom',
     defaultValue: false,
     validation: validate.isBoolean,
+    isWriteable: true,
   }, {
     name: 'integrationFolder',
     defaultValue: 'cypress/integration',
     validation: validate.isString,
     isFolder: true,
-  }, {
+  },
+  {
+    name: 'isInteractive',
+    defaultValue: true,
+    validation: validate.isBoolean,
+    isWriteable: true,
+  },
+  {
+    name: 'keystrokeDelay',
+    defaultValue: 0,
+    validation: validate.isNumberOrFalse,
+    isWriteable: true,
+  },
+  {
     name: 'modifyObstructiveCode',
```

Comment:
According to the list in the PR description, `modifyObstructiveCode` should be read-only.

Reply:
We change it with `Cypress.config()` here: https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/integration/issues/2582_spec.js#L8, so I think it is supposed to be writeable. I can update the PR description

Reply:
It works for that case, where we modify `window.frameElement`, but when we [use it in the proxy for modifying responses](https://github.com/cypress-io/cypress/blob/5e2de0c9ce3bfd4b3561fcb91883f52886c8dc4f/packages/proxy/lib/http/response-middleware.ts#L264), the runtime change won't carry through. I still think it should be read-only. We either need to use our internal exception for this, or move that test into a system test. We're only modifying it at runtime in that test for convenience's sake.